pygac/tests/test_reader.py

```diff
@@ -192,9 +192,50 @@ def test_get_tle_lines(self, read_tle_file, *mocks):
                 self.assertEqual(tle1, tle_data[tle_idx])
                 self.assertEqual(tle2, tle_data[tle_idx + 1])
 
+    @mock.patch('pygac.gac_reader.GACReader._get_corrupt_mask')
+    def test_get_angles(self, get_corrupt_mask):
+        """Test get_angles function of the reader."""
+        # Line: 1, 649, 6198 and 12658 from Tiros-N file (1980-01-03 11:47)
+        lon_i = np.array(
+            [69.41555, 152.10587, 164.3131, 67.23855])[:, np.newaxis]
+        lat_i = np.array(
+            [71.6283, 85.24265, -62.076958, 82.72296])[:, np.newaxis]
+        get_corrupt_mask.return_value = np.isnan(lon_i)
+        self.reader.lons = lon_i
+        self.reader.lats = lat_i
+        self.reader.tle_lines = [
+            '1 11060U 78096A   80003.54792075  .00000937  00000-0  52481-3 0  2588\r\n',
+            '2 11060  98.9783 332.1605 0012789  88.8047 271.4583 14.11682873 63073\r\n']
+        self.reader.utcs = np.array(
```

Comment:
You can also use mock to patch the return value of `GACReader.get_times`, `GACReader.get_lonlat` and `GACReader.get_tle_lines`

Reply:
I started by using mock to patch more return values. But then I got some problem and removed them. Should I update to use mock?